get.md

```diff
@@ -28,14 +27,13 @@ Many distributions offer LXDE out of the box, such as LXLE, openSUSE, and Debian
   <li><a href="https://get.opensuse.org" class="button special">Get openSUSE</a></li>
 </ul>
 
-
 #### I have a Linux or BSD running on my computer, and I want to try LXDE
 
 Sure. You can install LXDE easily. Please follow the wiki page to find out how to install LXDE
```

Comment:
So you think we could do a list of distros in 2024 have LXDE currently available? And if they have for example openSUSE instead of linking to the [homepage](https://get.opensuse.org/desktop/), link instead to their [docs](https://en.opensuse.org/Portal:LXDE), or in the case of Fedora directly to their [LXDE page](https://fedoraproject.org/spins/lxde/)?

Reply:
Personally I don't see why we should create such a list. I think every Linux user knows their own package manager and can just query it to see if lxde is there.
Such a list will also need to be updated and maintained. If you prefer a list you could use repology as well. For example: https://repology.org/project/lxde-common/versions or https://repology.org/badge/vertical-allrepos/lxde-common.svg
I would just remove the whole part and let users search for it in their distro.